src/models/__tests__/message_view.test.ts

```diff
@@ -194,6 +194,14 @@ describe("message_view", () => {
       );
     }
   });
+
+  it("GIVEN a valid message_view without third party object WHEN the object is decode THEN the decode succeed", async () => {
+    const result = MessageView.decode({
+      ...aMessageView,
+      components: { ...aComponents, thirdParty: undefined }
+    });
```

Comment:
I'd prefer to strip thirdParty property away

```suggestion
    const { thirdParty, ...componentsWithoutThirdParty } = aComponents;

    const result = MessageView.decode({
      ...aMessageView,
      components: componentsWithoutThirdParty
    });
```

Reply:
much better, thank you!